### ISSUE TITLE
可以新增 @cacheable, @cache_put, @cache_evict 吗

### ISSUE THREAD

**Issue by mkdir700** [long2ice/fastapi-cache#96]
`@cacheable`，用于绑定在查询方法上，可以根据查询结果判断是否进行缓存
`@cache_put`，用于绑定在新增方法上
`@cache_evict`，用于绑定在删除或更新方法上

参考：https://juejin.cn/post/6997440726627778597

**Comment by mkdir700** [comment]
> What's the status of this issue, and could I request an english version?

I'm very sorry, I have now updated this issue in English.

**Comment by mjpieters** [comment]
Thanks for updating, the summaries are really helpful!

I can kind-of see this work for FastAPI-Cache too, provided the arguments are compatible. It's something to think about for a post-v1.0.0 version.

**Comment by mkdir700** [comment]
Yes, we can consider whether to implement it after version V1.0.0

**Comment by vithu30** [comment]
is this feature added in newer release?
